test/test_runner_battle.c

```diff
@@ -284,16 +284,17 @@ static void BattleTest_Run(void *data)
         break;
     }
 
-    for (i = 0; i < STATE->battlersCount; i++)
+    for (i = 0; i < MAX_LINK_PLAYERS; i++)
```

Comment:
```suggestion
    STATE->battlersCount = STATE->battlersCount > MAX_LINK_PLAYERS ? STATE->battlersCount :  MAX_LINK_PLAYERS
    for (i = 0; i < MAX_LINK_PLAYERS; i++)
```

Probably doesn't matter that much but would avoid a second loop, just a suggestion (non blocking).